software_reseller/data/product_template.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <data noupdate="1">
+        <record id="software_reseller.product_template_1_f06ecb8a" model="product.template" context="{'create_product_product': False}">
```

Comment:
is it possible to remove the 'hash' at the end of the xml_id?

Reply:
@pimodoo yes, but then we need to add some test cases or something that notifies the user that the same XML ID is found plz double-check it.
and overwrite it with some commands in GitHub like security.

Reply:
as the xml_id contains the module name, there should not be two times the same xml_id. It is exactly the same for any normal Odoo module.

Reply:
okay

Reply:
then we create some tools using regular expiration that delete this `hash`

Reply:
It's the standard behaviour of `export_data`, `__ensure_xml_id`. They'll patch the standard code to improve that.